src/climatevision/generator/refdata.py

```diff
@@ -121,6 +121,25 @@ def load(
     def rows(self) -> Iterable[tuple[KeyT, list[str]]]:
         return self._rows.items()
 
+    def columns(self, keyword_data: str) -> tuple[str, dict[str, str]]:
+        """Return column of a given keyword."""
+        column_dict = {"": ""}
+        column_dict.update()
+        idx_header = self.header.get(keyword_data, 0)
+        for row in self._rows.items():
+            upd = {str(row[0]): str(row[1][idx_header])}
+            column_dict.update(upd)
+        return (keyword_data, column_dict)
+
+    def column_ags(self) -> list[str]:
+        """Return list of all ags in dataframe except germany"""
+        lst = [""]
```

Comment:
Why are you adding a "" to the list at the beginning. Removing it at the end is relatively expensive because it will a) iterate through everything and b) copy everything from later in the underlying array to earlier.  Just to `lst = []` (or maybe `lst: list[str] = []`

Reply:
Ja, manchmal macht man komische Sachen, weil andere nicht funktionieren... Und dann vergisst man sie :D